Sources/App/Controllers/API/API+PackageController.swift

```diff
@@ -3,37 +3,65 @@ import Vapor
 
 
 extension API {
-
+    
```

Comment:
Did you turn off the trim whitespace setting in Xcode? I've not seen it insert whitespace like this in your PRs before.

Reply:
I re-indented with the new Xcode-12 formatting. I've just checked and "Automatically trim ..." is set. I think this wasn't properly trimmed previously! All good :)

Reply:
Interesting. In Xcode 12 beta with these settings:

<img width="375" alt="Screenshot 2020-06-28 at 19 43 04@2x" src="https://user-images.githubusercontent.com/5180/85955604-9a591e80-b977-11ea-8d93-ec135390f757.png">

When I save this file, 91051cc happens.

Reply:
Ah, I've only had the first one ticked. I'll tick the other one as well then.